Makefile

```diff
@@ -53,10 +53,10 @@ generate_schema:: tfgen
 
 provider::
 	go generate ${PROJECT}/provider/cmd/${PROVIDER}
-	cd provider && go install -ldflags "-X github.com/pulumi/pulumi-azure/provider/pkg/version.Version=${VERSION}" ${PROJECT}/provider/cmd/${PROVIDER}
+	cd provider && go install -ldflags "-X github.com/pulumi/pulumi-azure/provider/v2/pkg/version.Version=${VERSION}" ${PROJECT}/provider/cmd/${PROVIDER}
```

Comment:
Probably worth checking the version is propagated correctly here.

Reply:
It is not - it *should* be but it's not :/

Reply:
ok, fixed :)